### PR TITLE
Fixes and adjustments

### PR DESCRIPTION
**Pull request by kevmodrome**

### CODE REVIEW

File: src/pages/recipes/_layout.svelte [hunk 1]
```diff
@@ -1,96 +1,31 @@
 <script>
   import CategoryTree from "@/components/recipes/CategoryTree.svelte";
```
Comment (peopledrivemecrazy) [inline src/pages/recipes/_layout.svelte]: This was never used in this component

File: src/pages/recipes/_layout.svelte [hunk 2]
```diff
@@ -1,96 +1,31 @@
...
-  import { setContext } from 'svelte'
+  import { setContext } from "svelte";
 
   import { layout, page } from "@roxi/routify";
```
Comment (peopledrivemecrazy) [inline src/pages/recipes/_layout.svelte]: page also is unused

**Review by peopledrivemecrazy — CHANGES_REQUESTED** [review]
Looks good 👍

**Review by peopledrivemecrazy — APPROVED** [review]

**Comment by diepoe** [comment]
Please wait with mergin, I'll push the fixes soon

File: src/pages/recipes/_layout.svelte (hunk 2 above)
Comment (diepoe) [reply src/pages/recipes/_layout.svelte]: this is actually used in the `index.svelte` file

**Pull request merged**